src/Data/String/LineWrapping.purs

```diff
@@ -40,6 +40,12 @@ newtype MeasuredWord = MeasuredWord MeasuredWordR
 
 newtype WrappedLine = WrappedLine (Array MeasuredWord)
 
+instance eqMeasuredWord ∷ Eq MeasuredWord where
+  eq (MeasuredWord x) (MeasuredWord y) =
+    (x.string == y.string) && (x.width == y.width)
```

Comment:
(I'm hoping you just hadn't tried it, since it's a bug in the compiler otherwise :cry:)